app.js

```diff
@@ -38,6 +60,59 @@ socket = io.listen(server, {
     path: BASE_URI + '/socket.io'
 });
 
+// thank you https://regex101.com/
+//
+// here are example urls:
+//
+// url: /api/v1/fs/Users/efranz/Downloads/
+// path: /Users/efranz/Downloads/
+// rx: ^\/api\/v1\/fs(.*)
+//
+// url: /api/v1/fs/Users/efranz/Downloads/fascinating.txt?download=1544841048995
+// path: /Users/efranz/Downloads/fascinating.txt
+// rx: ^\/api\/v1\/fs(.*)
+//
+// url: /api/v1/fs/Users/efranz/Downloads/IT7.5.1-WIAG.txt?hash
+// path: /Users/efranz/Downloads/IT7.5.1-WIAG.txt
+// 
+// url: /oodzip/Users/efranz/Downloads/gs
+// path: /Users/efranz/Downloads/gs
+//
+// url: /fs/Applications/
+// path: /Applications/
+if(whitelist.enabled()) {
```

Comment:
From this context `whitelist.enabled()` is never `true`. Instead of the middleware being conditionally defined we should have an early exit inside the lambda.

Reply:
That is by design. `whitelist.enabled()` is true only when WHITELIST env is set. Oops, its supposed to be WHITELIST_PATH. Will fix that.